apps/virtual_joystick/requirements.txt

```
@@ -0,0 +1,2 @@
+kivy
+git+https://github.com/farm-ng/amiga-brain-api@canbus-packet
```

After merging https://github.com/farm-ng/amiga-brain-api/pull/18
```suggestion
git+https://github.com/farm-ng/amiga-brain-api@main
```